src/program.ts

```diff
@@ -0,0 +1,49 @@
+import {
```

Comment:
File name is a little too generic.

Reply:
I don't think it is: we have the store, the hooks, the types, the web worker and we have the zk program